packages/tools/pie-css/test/css/__snapshots__/css.spec.ts.snap

```diff
@@ -287,6 +287,13 @@ exports[`index.css > should render the expected CSS content 1`] = `
     --dt-font-weight-extrabold: 800;
     --dt-font-weight-regular: 400;
     /* Alias tokens - Font */
+    /* Heading-xs font theme */
+    --dt-font-heading-xs-size--wide: var(--dt-font-size-16);
+    --dt-font-heading-xs-size--narrow: var(--dt-font-size-14);
+    --dt-font-heading-xs-family: var(--dt-font-family-primary);
+    --dt-font-heading-xs-weight: var(--dt-font-weight-extrabold);
+    --dt-font-heading-xs-line-height--wide: var(--dt-font-size-16-line-height);
+    --dt-font-heading-xs-line-height--narrow: var(--dt-font-size-14-line-height);
```

Comment:
do we really need a variable if the number is included in it?

Can we simplify and reduce dist code?

Reply:
Good question. The numerical tokens are global tokens, and the ones on the left-hand side are alias tokens. Global tokens shouldn’t be used directly by consumers, as they are considered raw values and are prone to change. Instead, consumers should rely on semantic alias tokens such as heading-xs, heading-sm, etc. This way, if the value of `heading-xs` changes to something else, such as size-12, users will automatically receive the update and don't have to manually update their implementation. 

This is a common practice with design tokens, and you can find more details about the approach here: https://pie.design/foundations/design-tokens/